### ISSUE TITLE
System.Windows.Input.ModifierKeysConverter.GetModifierKeys - NotSupportedException on Preview6

### ISSUE THREAD

**Issue by Symbai** [dotnet/wpf#1033]
* .NET Core Version: 3.0 Preview6
* Windows version: 10
* Does the bug reproduce also in WPF for .NET Framework 4.8?: No
* Does the bug reproduce also in WPF for .NET Core Preview 4: No

 **Problem description:**

`<MenuItem Command="ApplicationCommands.Undo"  />` crashes with `System.NotSupportedException: Unknown ModifierKeys "STRG".` While STRG is the German translation of CTRL. Switching back to Preview4 and everything works fine.

Please note that ApplicationCommands.Copy and others, I assume all that uses CTRL, crashes too.

 
 **Actual behavior:** 

```
"   at System.Windows.Markup.XamlReader.RewrapException(Exception e, IXamlLineInfo lineInfo, Uri baseUri)"
```

  **Minimal repo:** 
You **may** need to run this with a German Windows version.
```
<Window
    x:Class="WpfApp3.MainWindow"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:local="clr-namespace:WpfApp3"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006">
    <Grid>
        <Menu>
            <MenuItem Command="ApplicationCommands.Undo" />
        </Menu>
    </Grid>
</Window>
```
[WpfApp3.zip](https://github.com/dotnet/wpf/files/3311359/WpfApp3.zip)

**Comment by weltkante** [comment]
This was already fixed by #919 - you can use daily builds as described [here](https://github.com/dotnet/core/blob/master/daily-builds.md) or change your resource language to en-US (`CultureInfo.CurrentUICulture = CultureInfo.GetCultureInfo("en-US");`) until preview 7 is released.

**Comment by Symbai** [comment]
Thanks!
